.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: Continous Integration
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+
+env:
+  CI: true
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```

Comment:
Is it possible to check on ubuntu-20.x ?

Reply:
pls see comment below.